src/main/resources/application.yml

```diff
@@ -20,10 +20,79 @@ management:
 api:
   context-path: "/api"
 
+security:
+  basic:
+    enabled: \${SECURITY_ENABLED:true}
+  user:
+    name: "admin"
+    password: "admin"
```

Comment:
I did have a question about the `security.user.name` and `.password` properties.  By moving the configuration into the embedded application.yml, those WILL be the values in any environment unless overridden.  Are those values overridden in SBAC's production/staging environments?  I believe the default is to pick a random alphanumeric generated at runtime.

Reply:
I think we'll need to pull this out or update the deployment files for all environments

Reply:
Agreed, removed.  If somebody wants to set it locally, they can do so via runtime environment variables, java properties, docker-compose environment variables, etc, etc